resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -327,22 +327,38 @@ function task_preinst_deb {
 <% end -%>
 }
 
-# Debian based post-installation tasks.
+# Debian based unconditional post-installation tasks.
 function task_postinst_deb {
     task postinst_permissions
 <% EZBake::Config[:debian][:additional_postinst].each do |cmd| -%>
     <%= cmd %>
 <% end -%>
 }
 
-# RPM based post-installation tasks.
+# Debian based install post-installation tasks.
+function task_postinst_deb_install {
+    task postinst_permissions
```

Comment:
Will calling postinst_permissions twice cause problems?

Reply:
oh yeah we should drop that. copy-pasta